nextcord/message.py

```diff
@@ -362,9 +372,22 @@ def to_dict(self) -> AttachmentPayload:
             result["content_type"] = self.content_type
         if self.description:
             result["description"] = self.description
+        if self.duration_secs:
+            result["duration_secs"] = self.duration_secs
+        if self._waveform:
+            result["waveform"] = self._waveform
         return result
 
     @property
+    def waveform(self) -> Optional[array[int]]:
+        """Optional[array[:class:`int`]]: The base64 decoded data representing a sampled waveform
+        (currently for voice messages).
+
+        .. versionadded:: 3.0
+        """
+        if self._waveform is not None:
+            return array("B", [int(b) for b in base64.b64decode(self._waveform)])
```

Comment:
This is not a cheap conversion, especially with more data. Make this an `utils.cached_property`